AUTHORS

```diff
@@ -43,4 +43,5 @@ Contributors are:
 -Liam Beguin <liambeguin _at_ gmail.com>
 -Ram Rachum <ram _at_ rachum.com>
 -Alba Mendez <me _at_ alba.sh>
+-Jeremy Retailleau <jeremy.retailleau _at_ gmail.com>
```

Comment:
Having an AUTHORS file was inherited from when I took over the project a decade ago. The contribution guidelines shouldn't force anyone to add themselves there, please feel free to remove yourself if it doesn't feel right.

The contribution guidelines have been updated to reflect that stance - I wasn't aware it actually demanded that, sorry for that.

Reply:
Thanks for the clarification. I'm gonna remove my name then as git history should be enough to find out who did what :)